api/cloud-control/v1beta1/labels.go

```diff
@@ -1,7 +1,15 @@
 package v1beta1
 
+// Used for K8s labels
 const (
 	LabelKymaName        = "cloud-manager.kyma-project.io/kymaName"
 	LabelRemoteName      = "cloud-manager.kyma-project.io/remoteName"
 	LabelRemoteNamespace = "cloud-manager.kyma-project.io/remoteNamespace"
 )
+
+// Used for Hyperscaler labels mainly backups which need to be nuked without KCP mirror object
+const (
+	ManagedByKey   = "managed-by"
```

Comment:
If this is not K8S related, but GCP scope only, it should not be in the `api` package but in the `gcp/nuke` package

Reply:
This can be used by gcp or any other provider as the label. And its usage theoretically is not restricted to nuke.
For now I will move it under kcp/provider/gcp/client/gcpConstants. If other providers want to use the exact same label, they can move it to somewhere more generic or they can have their own labels.